### DISCUSSION TITLE
When will be released the next tag?

### DISCUSSION THREAD

**Discussion by MirtoBusico** [apache/apisix#6678] [Q&A] [answered]
Hi all,
I need two features that where merged in master
What is the plan for releasing the next tag?

**Comment by leslie-tsang** [comment, marked as answer]
Hello there, [here](https://lists.apache.org/thread/mc6flhkskcbwh5q5dd6t3moj8d0k8w7y) is the voting thread, please check it out. :)

**Reply by MirtoBusico** [reply]
Thanks
